docs/pieces.md

```diff
@@ -380,4 +375,4 @@ Pieces often implement async components as an easy way to display them on the pa
 
 ## Piece pages
 
-As referred to above, while they are not needed in every case, [piece pages](piece-pages.md) are a crucial companion to pieces for any website that allows pieces to be browsed by the user. See the [piece pages](piece-pages.md) section to get started.
+As referred to above, while they are not needed in every case, [piece pages](piece-pages.md) are a crucial companion to pieces for any project that allows the user to browse an index of pieces. See the [piece pages](piece-pages.md) section to get started.
```

Comment:
`website` is correct here. piece pages are not typically relevant in headless projects. The developer would likely just use the REST API of their piece module.